storybook/src/elements/nuxeo-video/nuxeo-video-conversions.stories.js

```diff
@@ -0,0 +1,28 @@
+import { html } from 'lit-html';
+import { storiesOf } from '@storybook/polymer';
+import { text } from '@storybook/addon-knobs';
+import '@nuxeo/nuxeo-ui-elements/nuxeo-video/nuxeo-video-conversions';
+import DocumentBuilder from '../../data/documents.data';
+import videoProperties from '../../video/videoProperties.js';
+
+storiesOf('UI/nuxeo-video', module).add('Nuxeo Video Converter', () => {
+  const documentBuilder = new DocumentBuilder()
+    .setType('File')
+    .setProperties({ 'vid:transcodedVideos': videoProperties['vid:transcodedVideos'] });
+  documentBuilder.build();
+  const label = text('Label', 'Video Conversion');
+  return html`
+    <style>
```

Comment:
Is this style a copy of `nuxeo-card`?
We may want to apply it directly, no?  https://github.com/nuxeo/nuxeo-elements/blob/master/storybook/.storybook/nuxeo-demo-theme.js#L42-L50